### ISSUE TITLE
Change upload to use S3 instead of directly to the app

### ISSUE THREAD

**Issue by pdpinch** [mitodl/lore#113]
Courses can be quite large and during import the app on Heroku can time out. S3 does not timeout so we can use it for uploads. 

There is an integration to django-storages that already does this. 

It would be nice to have an option that isn't tied to S3 (such as one that supports Openstack Swift)

**Comment by pdpinch** [comment]
Courses can be quite large and during import the browser can time out. Uploading to S3 allows us to do an asynchronous file upload.

**Comment by pdpinch** [comment]
@carsongee this was addressed by #145 wasn't it?

**Comment by carsongee** [comment]
I don't think so.  I think this was intended for the client side upload since it mentions timeouts

**Comment by carsongee** [comment]
lol, was clearing out old e-mail and apparently that used to be the title: `Upload courses via javascript to s3`
